src/__tests__/element-queries.js

```diff
@@ -66,4 +67,18 @@ test('totally empty label', () => {
   expect(() => getByLabelText('')).toThrowErrorMatchingSnapshot()
 })
 
+test('using jest helpers to assert element states', () => {
+  const {queryByTestId, getByTestId} = render(
+    <span data-testid="count-value">2</span>,
+  )
+
+  //other ways to assert your test cases, but you don't need all of them.
```

Comment:
Not sure why prettier didn't add a space here, but please add a space before the `o` :)